### ISSUE TITLE
Feature vector of masked region

### ISSUE THREAD

**Issue by VellalaVineethKumar** [facebookresearch/maskrcnn-benchmark#696]
## 🚀 Feature

The feature vector of the masked region

## Motivation
i was thinking of using this feature vector for data association in tracking so wanted to experiment and see if i can manipulate things and get work done.

## Pitch

In the Jupyter notebooks provided i wanted to see the output of the feature vector of the masked region.

i would be much happier if i also get to know how to extract the feature vector of ROI bbox

**Comment by LeviViana** [comment]
I'm not sure we need a specific feature for this task.

You can get all feature maps just by exploring the detector's pipeline. For instance, the following snippet can give you a grasp on how to do it:
```python
from maskrcnn_benchmark.structures.image_list import to_image_list
t = coco_demo.transforms(image)
t = to_image_list(t, coco_demo.cfg.DATALOADER.SIZE_DIVISIBILITY)
# Getting all backbone's feature maps
for x in coco_demo.model.backbone[0](t.tensors):print(x.size())
# Getting FPN's feature maps
for x in coco_demo.model.backbone[1](coco_demo.model.backbone[0](t.tensors)):print(x.size())
# Plotting a feature map
feat_map = coco_demo.model.backbone[1](coco_demo.model.backbone[0](t))[2]
plt.imshow(feat_map[0, 1, :, :].detach().numpy())
plt.show()
# Getting feature maps of masks:
box_list = coco_demo.model(t.tensors)
mask_map, _, _ = coco_demo.model.roi_heads.mask(
                    coco_demo.model.backbone[1](coco_demo.model.backbone[0](t.tensors)), 
                    box_list
                )
plt.imshow(mask_map[3, 0, :, :].detach().numpy())
plt.show()
```
Try running parts of this snippet in the Jupyter Notebook provided in the demo folder (paste it below the 8th cell).
